### PR TITLE
write data to raw DB

### PR DESCRIPTION
**Pull request by jemtech**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
  # installs dependencies from ./setup.py, and the package itself, in editable mode
-.
+.pymongo
```
Comment (top-on) [inline requirements.txt]: we're keeping our dependencies in setup.py, currently line 14.
Would you mind adding it to the list there?

File: scripts/divi-icu-beds.py [hunk 2]
```diff
@@ -50,6 +52,8 @@
 meta["unix_timestmap"] = unix_utc
 meta["iso_timestamp"] = iso_utc
 
+mongoDBUpload.insertData(states, meta)
```
Comment (top-on) [inline scripts/divi-icu-beds.py]: I would be afraid of this script to fail if the mongoDB is not accessible.
how about you move all mongo-related things to scripts/mongoDBUpload.py ?
there you could e.g. read in the (alphabetically) latest JSON in data and upload then.
Then we could schedule that mongo action after the autocommit.

How does that sound to you? We're open for discussion :-)

**Review by top-on — CHANGES_REQUESTED** [review]
Highly appreciated!
I left some comments for coupling the 'mongo action' a bit more loosely, and after scraping and autocommit.
As said, we're open for discussion and other ideas :-)

And I assume your current changes get you the data to MongoDB?

File: requirements.txt (hunk 1 above)
Comment (jemtech) [reply requirements.txt]: will do so

File: scripts/divi-icu-beds.py (hunk 2 above)
Comment (jemtech) [reply scripts/divi-icu-beds.py]: i just thought we would reduce overhead of parsing hear

Comment (jemtech) [reply scripts/divi-icu-beds.py]: would be try catch ok ?

Comment (top-on) [reply scripts/divi-icu-beds.py]: ok for me, thanks

**Comment by top-on** [comment]
Closing this PR, but including its commits plus a few additional ones in PR #2. 
@jemtech : could you please test if the upload to Mongo still works for you on PR #2 ?

**Pull request closed without merging**